lookout/style/typos/preprocessing.py

```diff
@@ -28,3 +30,22 @@ def filter_splits(data: pandas.DataFrame, tokens: Set[str]) -> pandas.DataFrame:
     :return: Filtered dataframe.
     """
     return data[data[Columns.Split].apply(lambda x: check_split(x, tokens))]
+
+
+def print_frequencies(tokens: Set[str], id_stats: pandas.DataFrame,
+                      frequency_column: str, path: str) -> None:
+    """
+    Print frequencies of tokens to a file.
+
+    Frequencies info is obtained from id_stats dataframe.
+    :param tokens: Set of tokens, for which frequencies should be printed.
+    :param id_stats: Dataframe with frequency information for tokens. \
+                     It must be indexed by tokens and contain column `frequency_column`.
+    :param frequency_column: Name of the column in `id_stats`, from which to take frequency info.
+    :param path: Path to a .csv file to print frequencies to.
+    """
+    frequencies = id_stats.loc[tokens].dropna()[frequency_column]
+    with smart_open(path, "w") as f:
+        writer = csv.writer(f, delimiter=" ")
```

Comment:
well, may let's use a regular delimiter? 
Also, it is a good idea to add the header.

Reply:
I only leave `" "` because original symspell works with them (in case we need to preserve the compatibility with it). If it doesn't matter, I will change to the comma.

Reply:
ok, let's keep `" "`

Reply:
@irinakhismatullina Since you are going to replace the implementation of load_dictionary in symspell in #617, it indeed makes sense to use a comma.